src/common/chat/Tokenizer.ts

```diff
@@ -11,6 +11,7 @@ import { Regex } from "@/site/twitch.tv";
 import { parse as tldParse } from "tldts";
 
 const URL_PROTOCOL_REGEXP = /^https?:\/\//i;
+const URL_MESSAGE_CLEANUP_REGEXP = /^[\s.,;()[\]]*|[\s.,;()[\]]*$/g;
```

Comment:
Instead of running the message through two regexes you could change `URL_PROTOCOL_REGEXP` to match whole URLs.

Reply:
Hello. I'm not very good at regulars. If you know how these two expressions can be combined, please tell me.

Reply:
There are great resources on the Internet. Search for a regex that matches URLs in a string.